src/main/frontend/components/PizzaStyleShow.js

```diff
@@ -1,22 +1,44 @@
 import React, { useEffect, useState } from "react"
 import { Link, useLocation } from "react-router-dom"
+import _ from "lodash"
 import ReviewTile from "./ReviewTile"
-import PizzaStyleTile from "./PizzaStyleTile"
 import { jsonGet, jsonDelete } from "../public/js/jsonFetch"
 
 const PizzaStyleShow = props => {
   const [pizzaStyle, setPizzaStyle] = useState({ reviews: [] })
+  const [errors, setErrors] = useState("")
   let location = useLocation()
 
   const fetchPizzaStyle = async () => {
-    const respBody = await jsonGet(`/api/v1/pizza-styles/${props.match.params.id}`)
-    setPizzaStyle(respBody.pizzaStyle)
+    try {
+      const response = await fetch(`/api/v1/pizza-styles/${props.match.params.id}`)
+      if (!response.ok) {
+        if (response.status === 422) {
+          const body = await response.json()
+          console.log(body)
+          setPizzaStyle({})
+          return setErrors(body.message)
+        } else {
+          const errorMessage = `${response.status} (${response.statusText})`
+          const error = new Error(errorMessage)
+          throw error
+        }
+      }
+      const pizzaStyleData = await response.json()
+      setPizzaStyle(pizzaStyleData.pizzaStyle)
+    } catch (err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
```

Comment:
Just to clarify this is getting rid of the abstracted `jsonGet` method and this is intended?

Reply:
Yes, this was intended but may be refactored later